cmd/cleanproto/Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: build test
 
 build:
-	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o cleanproto .
+	CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -o cleanproto .
```

Comment:
Can we remove all those flags?

I think those flags were for local build and copying into docker container.
And then copied from the bns makefile to here.

Let's remove more flags if it works on mac